### PR TITLE
Send download link and rename summary agent

### PR DESCRIPTION
**Pull request by ashish-spext**
- Sending the download link through agent instead of relying on RE solves https://github.com/video-db/Director/issues/73
- Improved name of summary agent from `video_summary` to `summarize_video`

### CODE REVIEW

File: backend/director/agents/download.py [hunk 1]
```diff
@@ -30,13 +28,33 @@ def run(
...
+            if download_response.get("status") == "done":
+                download_url = download_response.get("download_url")
+                name = download_response.get("name")
+                text_content.text = (
+                    f"<a href='{download_url}' target='_blank'>{name}</a>"
+                )
+                text_content.status = MsgStatus.success
+                text_content.status_message = "Here is the download link"
+                self.output_message.publish()
+            else:
```
Comment (ankit-v2-3) [inline backend/director/agents/download.py:46]: Can we also change the `text_content` to error before returning error `AgentResponse` ?

Comment (ashish-spext) [reply backend/director/agents/download.py:46]: Pushed, can you test that condition now? it should set the error state

Comment (ankit-v2-3) [reply backend/director/agents/download.py:46]: yeah, working now.

Screenshot 2024-11-25 at 11 48 42 AM

**Review by ankit-v2-3 — APPROVED** [review]
looks good to me.

**Pull request merged**